.github/workflows/ci.yml

```diff
@@ -7,22 +7,22 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        project: ['.', 'glfw', 'freetype']
+        project: [".", "glfw", "freetype"]
     # We want to run on external PRs, but not on our own internal PRs as they'll be run by the push
     # to the branch.
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
     env:
-      DISPLAY: ':99.0'
+      DISPLAY: ":99.0"
```

Comment:
It's a bit unfortunate we now have different formatting for CI files throughout this repository.

I suggest we add a new script: `dev/ci-lint.sh` which runs:

* actionlint: https://github.com/rhysd/actionlint/blob/main/docs/install.md
* Some YAML formatter.. preferable not one from the npm ecosystem, maybe this Go one https://github.com/UltiRequiem/yamlfmt

on all our CI yml files.

Reply:
(I can address this in a future PR)

Reply:
`yamlfmt` results the same output. at least im gonna say it has nothing to do with quotes `"`, `'`

Reply:
To be clear, I am fine with the formatting being anything (double quotes, whatever) as long as we have a way for it to be consistent. We can use yamlfmt.

Reply:
i think **actionlint** is enough and yaml formatters doesn't make any change as long as we just edit github workflows. that's why there's no many yaml formatters, or at least they are unused in most cases

Reply:
https://github.com/hexops/mach/issues/430